### ISSUE TITLE
[portal] Better a11y support

### ISSUE THREAD

**Issue by tapiiron** [gravitee-io/issues#3995]
In the EU-directive web pages should be made accessible. Have you concidered WCAG 2.1 compliance?

https://www.w3.org/WAI/WCAG21/Understanding/

### Your Environment
* Version used: 3.0.3

**Comment by aelamrani** [comment]
Yes we do. 

Do you see anything that is not conform to the accessibility?

**Comment by tapiiron** [comment]
Seems like the list of API:s in default catalog can not be targeted by keyboard only. Only way to get to the wanted api is by search and the selecting the one needed from list.
1. <Tab> into Catalog and press <Enter>
2. Try <Tab> into one api in generated in the list below the first highlighted api
3. Notice that you can't tab into list items

**Comment by aelamrani** [comment]
Ok thanks for reporting that one, we will have a look to this.
